backend/src/db/migrations/20230909170408_change_meditation_entries.ts

```diff
@@ -0,0 +1,40 @@
+import { type Knex } from 'knex';
+
+const TableName = {
+  MEDITATION_ENTRIES: 'meditation_entries',
+  MEDITATION_TOPICS: 'meditation_topics',
+};
+
+const ColumnName = {
+  ID: 'id',
+  TOPIC_ID: 'topic_id',
+  AUDIO_URL: 'audio_url',
```

Comment:
we may store video here in the future too, also add content_type as for files table
```suggestion
  MEDIA_URL: 'media_url',
```

Reply:
Done